hasura/metadata/cron_triggers.yaml

```diff
@@ -1,35 +1 @@
-- name: check-nominee
```

Comment:
@sshmm revert changes to this file pelase.

Reply:
done. I have to do it on multiple commits because VS code formatter add extra spaces when I save the file manually